githubteacher.md

```diff
@@ -0,0 +1,6 @@
+# GitHub Teacher
+
+* Name: Joshua Wehner
+* Username: jaw6
+* Location: Minneapolis, MN
+* Breakfast: Oatmeal
```

Comment:
@githubteacher that sounds so dry... did you eat it with milk or water? Please be as precise as possible.